modules/post/windows/gather/forensics/memorydump.rb

```diff
@@ -0,0 +1,230 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+#
+# This Module has an embred Belkasoft RAM Capturer version of 2018-08
```

Comment:
Does it? I'm not sure if the Belkasoft license allows packaging it, however the PR doesn't package it anyway...